inc/com/centreon/engine/flapping.hh

```diff
@@ -39,13 +39,13 @@ void enable_flap_detection_routines();
 // disables flap detection on a program-wide basis
 void disable_flap_detection_routines();
 // enables flap detection for a particular host
-void enable_host_flap_detection(com::centreon::engine::host* hst);
+void enable_host_flap_detection(std::shared_ptr<com::centreon::engine::host> hst);
```

Comment:
This function could become a method and moved into host class.
As we should also have service with a similar one, I can expect that this code is moved into notifer class.